scripts/apps/archive/views/preview.html

```diff
@@ -46,6 +46,12 @@
               <span ng-if="item._type === 'archived'" class="stage" tooltip="{{:: 'Archived from' | translate}} {{deskName}} / {{ stage }}" tooltip-placement="bottom" ng-show="stage"><b>{{:: 'Archived from' | translate}} {{deskName}}</b> / {{ stage }}</span>
               <span ng-if="item._type !== 'archived'" class="stage" tooltip="{{deskName}} / {{ stage }}" tooltip-placement="bottom" ng-show="stage"><b>{{deskName}}</b> / {{ stage }}</span>
             </div>
+            <div class="preview-header__flex-row preview-header__flex-row--single-line" ng-if="subjectPreviewFields.length">
+              <div class="tag-label" ng-repeat="author in selected.preview.authors">{{author.role}} : {{author.sub_label}}</div>
+            </div>
+            <div class="preview-header__flex-row preview-header__flex-row--single-line" ng-if="checkDuplicateKeywords">
+              <div class="tag-label" ng-repeat="keyword in checkDuplicateKeywords">{{keyword}}</div>
```

Comment:
does it call the function automatically?

Reply:
Yes it is